src/components/ArticleDetail.jsx

```diff
@@ -16,8 +17,22 @@ function ArticleDetail({ singleArticle }) {
     });
   }, []);
 
+  function handleUpvote(articleID) {
+    setSingleArticle({ ...singleArticle, votes: singleArticle.votes + 1 });
+
+    upVoteArticle(articleID)
+      .then((updatedArticle) => {
+        setSingleArticle(updatedArticle);
+      })
+      .catch((err) => {
+        if (err) {
+          setErr("Something went wrong");
```

Comment:
I like that this is there, to the user the number goes up then they see the message. Is there a way to perhaps either reset the number when this happens, or avoid it going up?